routes/meeting_router.js

```diff
@@ -1,20 +1,24 @@
 const {meetingService} = require("../services/meeting_service")
-var express = require('express');
-var router = express.Router();
+const express = require('express');
+const router = express.Router();
 
-// TOMEK
-router.get('/<UUID_meeting>', function(req, res, next) {
-  res.send("Timeslots from hosts");  
+router.get('/:uuid', function(req, res, next) {
+  const uuid = req.params.uuid
+  meetingService.getTimeSlotsIntersection(uuid)
+      .then(result => res.status(result.status).send(result))
 });
-router.put('/<UUID_meeting>/pick_time_slot', function(req, res, next) {
-  res.send("choose timeslot");  
+router.put('/:uuid/pick_time_slot', function(req, res, next) {
+  const uuid = req.params.uuid
+  const parsed = req.body
+  meetingService.pickTimeSlot(uuid, parsed)
+      .then(result => res.status(result.status).send(result))
```

Comment:
Takie pytanie - tu ustawiamy status, wcześniej (w tym pr) - nie. To się jakoś automatycznie robi?

Reply:
gdzie? (poza postem na hostach, który jak mówiłem jest do wywalenia XD)

Reply:
Tak z pamięci Ci nie napiszę, ale wydaje mi się, że gdzieś tak było